docs/sphinxext/mantiddoc/directives/properties.py

```diff
@@ -75,15 +75,34 @@ def _create_properties_table(self):
 
             for prop in alg_properties:
                 # Append a tuple of properties to the list.
-                properties.append(
-                    (
-                        str(prop.name),
-                        str(direction_string[prop.direction]),
-                        property_type_dict.get(str(prop.type), str(prop.type)),
-                        str(self._get_default_prop(prop)),
-                        self._create_property_description_string(prop),
+                for prop in alg_properties:
```

Comment:
Why do you have an inner loop over all properties?

Reply:
Unfortunate copy-paste mistake. Thought it will be faster than re-base.